Bhalani-Mauli/restaurant-booking
# Create restaurant page

**What is this change doing?**: Here I am creating a UI + API + database integration for `adding new restaurants` page.
**How can I test it?**: 
- You can open /restaurants/create
- You can fill all the details required
- We also have validation
- and then you will see success screen.
- Still next success page is not done yet so I am just sending an html as a success

#### Checklist

- [x] tested it locally
- [x] used env file for token and other secrets
- [x] followed best practices and clean code
  - good naming conventions
  - single responsiblity (SOLID)
  - clean and redable code
  - write scalable code
